### ISSUE TITLE
Movie shows as available on plex, even when it is not

### ISSUE THREAD

**Issue by mecampbe02** [tidusjar/Ombi#2790]
**Describe the bug**
Movie is being listed as available on Plex.  Movie is not on my server.  Movie is on a server that is shared with me on another Plex Server.  My users don't have access to that server.  

**To Reproduce**
1. Owner of Plex server needs to have Someone else sharing a library with them from another server 
2.  Log into  Ombi as a user that only has access to the Owner server
3.  Search for movie (one that is not on your server but on a server shared with you)
4.  Shows as available because it's on a server shared with the Plex Owner account

**Expected behavior**
after search by user, Movie/TV show is displayed with either request or available marker.  Available should only show if the item is available on the server Ombi is attached to. 

**Screenshots**
![ombi search](https://user-images.githubusercontent.com/21971067/52286494-4d197e80-2936-11e9-853f-e7b48f2e6307.png)
![plex server](https://user-images.githubusercontent.com/21971067/52286496-4d197e80-2936-11e9-89da-3911a3669f4a.png)
![plex 1](https://user-images.githubusercontent.com/21971067/52286495-4d197e80-2936-11e9-897f-f9477199ec74.PNG)
![plex shared server](https://user-images.githubusercontent.com/21971067/52286497-4d197e80-2936-11e9-8044-c075f18da239.PNG)

**Logs (Logs directory where Ombi is located)**
No Logs since it is not generating an error

**Desktop (please complete the following information):**
 - OS: Windows 10

**Ombi Version (please complete the following information):**
 - Version 3.0.4119
- Media Server : Plex
- Radarr Version 0.2.0.1293
- Sonarr Version [2.0.0.5301]

**Additional context**

**Comment by tidusjar** [comment]
Please fully fill in the template. Reply when you have and I'll reopen

**Comment by mecampbe02** [comment]
finished with the full template

**Comment by tidusjar** [comment]
Which server have you actually added in the Ombi Plex Settings?

**Comment by mecampbe02** [comment]
Turing is the server I added

**Comment by tidusjar** [comment]
So the server you have connected ombi to, thats your server without the content?

**Comment by mecampbe02** [comment]
Correct, the Server I'm hosting is without the content.

With my test user, whom only has access to my hosted server receives an error in plex saying there is an error loading the item.

with plex owner account it always takes me to the same  unrelated TV content on my server.
I specified tv content since i'm using the example which is a movie content search

**Comment by rcdailey** [comment]
I was just about to open another issue, but I'll upvote this one instead. I'm definitely seeing the same issue. Mine is for a movie called Captain Marvel. It's not in my Plex library. But when I click "View on Plex", it takes me to some random anime show in my library...

Checked the logs, don't have anything there.

Here is my version information:

Item | Description
-- | --
Version | 3.0.4256
Branch | master
Github | https://github.com/tidusjar/Ombi
Discord | https://discord.gg/Sa7wNWb
Reddit | https://www.reddit.com/r/Ombi/
Issues | https://github.com/tidusjar/Ombi/issues
Wiki | https://github.com/tidusjar/Ombi/wiki
OS Architecture | X64
OS Description | Linux 4.15.0-45-generic #48-Ubuntu SMP Tue Jan 29 16:28:13 UTC 2019
Process Architecture | X64
Application Base Path | /opt/ombi

**Comment by rcdailey** [comment]
@tidusjar Any feedback on this one? Bot is trying to close it but it's unresolved AFAIC

**Comment by tidusjar** [comment]
I do not know what the issue is since I cannot reproduce it. 

Have you checked to see if your machine Id correct in ombi?

**Comment by PhAzE-Variance** [comment]
I'm seeing the same thing. A movie says it's available, click on "view in plex" and i'm taken to a random TV show instead.  I have no way to correct this from the GUI.

**Comment by tidusjar** [comment]
Please upload some screenshots. If you can it might be worth querying the OmbiExternal.db to check if the IDs match up. The data is stored in the `PlexServerContent` table

**Comment by jordanbommelje** [comment]
I'm having the same issue. I only noticed because the movie was in the top suggestions and it was just released in theaters recently:

![Ombi-Showing-Available](https://user-images.githubusercontent.com/7672128/71778544-56afca00-2f7d-11ea-98ca-26d5328094cf.png)

When I click on "View in Plex," I'm taken to basically a 404 page:

![Plex-404](https://user-images.githubusercontent.com/7672128/71778552-819a1e00-2f7d-11ea-820d-78cad72e8aaf.png)

I've tried running a manual Sync and it persists. This is my only server running Plex. Unfortunately I'm running Ombi in a Docker container on my NAS which I'm having trouble actually accessing any files to provide the OmbiExternal.db. 

Let me know if there's anything else I can do to test.
